mobile/joseki/joseki.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,807 @@
+// !$*UTF8*$!
```

Comment:
This looks like this is compiled on the build process, please include in build process and add to .gitignore